### DISCUSSION TITLE
No module named 'rgbmatrix'

### DISCUSSION THREAD

**Discussion by rampar20** [riffnshred/nhl-led-scoreboard#432] [General]
I installed via the compiled 1.7.0 img file and I get this error when I try to run a test:

 **python3 src/main.py --led-gpio-mapping=adafruit-hat --led-brightness=60 --led-slowdown-gpio=2**
Traceback (most recent call last):
  File "/home/rick/nhl-led-scoreboard/src/main.py", line 4, in <module>
    from data.scoreboard_config import ScoreboardConfig
  File "/home/rick/nhl-led-scoreboard/src/data/scoreboard_config.py", line 1, in <module>
    from utils import get_file
  File "/home/rick/nhl-led-scoreboard/src/utils.py", line 1, in <module>
    from rgbmatrix import RGBMatrixOptions, graphics
ModuleNotFoundError: **No module named 'rgbmatrix'**

**Comment by falkyre** [comment]
By image do you mean this one:  https://github.com/falkyre/nhl-led-scoreboard-img/releases/tag/v1.7.0

So did you download that image and use the rpi imager or etcher to put it on an SD card?

**Comment by rampar20** [comment]
Thanks for the reply. I am an idiot. I didn't realize there was new
documentation, and was following an older one from github. Apologies for
wasting your time. It is working for me now.
